src/features/MoviePage/Content/MovieTile/styled.js

```diff
@@ -0,0 +1,171 @@
+import styled from "styled-components";
+
+export const Title = styled.header`
+    font-weight: 600;
+    font-size: 36px;
+    margin-bottom: 18px;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.phone}) {
+        font-weight: 500;
+        font-size: 16px;
+        margin-bottom: 0;
+    }
+`;
+
+export const Image = styled.img`
+    width: 312px;
+    height: 464px;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.phone}) {
+        width: 114px;
+        height: 169px;
+    }
+`;
+
+export const Content = styled.div`
+    @media(max-width: ${({ theme }) => theme.breakpoint.phone}) {
+        width: 114px;
+        height: 169px;
+    }
+`;
+
+export const Year = styled.div`
+    font-size: 22px;
+    margin-bottom: 16px;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.phone}) {
+        color: ${({ theme }) => theme.color.waterloo};
+        font-size: 13px;
+        margin-bottom: 4px;
+    }
+`;
+
+export const Production = styled.div`
+    font-size: 18px;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.phone}) {
+        font-size: 12px;
+    }
+`;
+
+export const Info = styled.span`
+    color: ${({ theme }) => theme.color.stormGrey};
+    margin: 0 11px 6px 0;
+    line-height: 28px;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.phone}) {
+        display: none;
+    }
+`;
+
+export const Tags = styled.ul`
+    list-style: none;
+    display: flex;
+    flex-wrap: wrap;
+    padding-left: 0;
+    margin: 8px -8px;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.phone}) {
+        margin: 4px -8px 20px;
+    }
+`;
+
+export const Tag = styled.li`
+    background: ${({ theme }) => theme.color.mystic};
+    font-size: 14px;
+    padding: 6px 16px;
+    margin: 10px 8px;
+    border-radius: 5px;
+        
+    @media(max-width: ${({ theme }) => theme.breakpoint.phone}) {
+        display: flex;
+        font-size: 10px;
+        padding: 3px 8px;
+        margin: 4px 0 4px 8px;
+    }
+`;
+
+export const TagMiddle = styled.span`
+    @media(max-width: ${({ theme }) => theme.breakpoint.phone}) {
+        order: 1;
+    }
+`;
```

Comment:
Jeżeli chodzi o Mulan - działa, ale dla innych filmów o innych tagach już raczej nie spełni swojej roli 😜

Reply:
Też tak uważam, cały czas myślę jak to wyświetlić ale na razie nie mam pomysłu.

Reply:
Na razie tak, a jak już się uda to się to uaktualni. No chyba, że Ty wiesz jak, he he.

Reply:
Niee, tylko patrzylem na necie czy jakos sie da to zrobić i znalazłem jakieś rozwiązania, które trzebaby przestudiować i spróbować jakoś u nas zaimplementować, ale to nie teraz 😛

Reply:
Przejrzałem jeszcze raz lekcję Flexa i Grida i nic na razie mi nie przychodzi do głowy.